users_ldap_mail/readme/USAGE.rst

```diff
@@ -0,0 +1,16 @@
+To use this module, you need to:
+
+#. Open Odoo in your browser
+#. Go to Settings
+#. Go to General Settings
+#. In The General Settings form go to LDAP Parameters
+#. In a LDAP Parameters item there are two new fields: mail and name, the name
+   These items will correspond with a new user that is created, when a user
+   logs in via LDAP in Odoo.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
```

Comment:
I think this can go out, as there's a badge already for it generated by the oca-gen-readme